cmd/proxy_get.go

```diff
@@ -0,0 +1,148 @@
+package main
```

Comment:
Let's not copy this code, it's going to be hard to maintain. Instead, we could carve out whatever needs to be reused into a pkg in `openservicemesh/osm` and import it here.

Reply:
Great idea - will work on the other side.